setup.py

```diff
@@ -413,19 +413,12 @@ def _quote_build_define(argument):
         if BUILD_OVERRIDE_BORING_SSL_ASM_PLATFORM
         else util.get_platform()
     )
-    LINUX_X86_64 = "linux-x86_64"
-    LINUX_ARM = "linux-arm"
-    LINUX_AARCH64 = "linux-aarch64"
-    if LINUX_X86_64 == boringssl_asm_platform:
-        asm_key = "crypto_linux_x86_64"
-    elif LINUX_ARM == boringssl_asm_platform:
-        asm_key = "crypto_linux_arm"
-    elif LINUX_AARCH64 == boringssl_asm_platform:
-        asm_key = "crypto_linux_aarch64"
-    elif "mac" in boringssl_asm_platform and "x86_64" in boringssl_asm_platform:
-        asm_key = "crypto_apple_x86_64"
-    elif "mac" in boringssl_asm_platform and "arm64" in boringssl_asm_platform:
-        asm_key = "crypto_apple_aarch64"
+    # BoringSSL's gas-compatible assembly files are all internally conditioned
+    # by the preprocessor. Provided the platform has a gas-compatible assembler
+    # (i.e. not Windows), we can include the assembly files and let BoringSSL
+    # decide which ones should and shouldn't be used for the build.
+    if "win32" not in boringssl_asm_platform:
```

Comment:
Looks like the problem is Python's docs are false (sigh) and it doesn't return `sys.platform` on Windows. Will try to fix.